apps/duplication/archive_fetch.py

```diff
@@ -101,7 +101,7 @@ def fetch(self, docs, id=None, **kwargs):
 
             desk = get_resource_service('desks').find_one(req=None, _id=desk_id)
             if desk and desk.get('default_content_profile') and not doc.get('macro'):
```

Comment:
I would remove this `and not doc.get('macro')` part so it gets set also if there is macro which doesn't set profile